test/time.js

```diff
@@ -3,6 +3,22 @@
 const metatests = require('metatests');
 const common = require('..');
 
+const DefaultDate = Date;
+const dateNowValue = 1561975200000; // 2019-07-01T10:00:00.000Z
+// eslint-disable-next-line no-global-assign
+Date = function(...args) {
```

Comment:
I don't like the idea of mutating global state, currently `metatests` doesn't isolate tests, this will lead to unexpected behavior in every other test.
Currently I see 2 ways to fix it:
* Add sandboxing to metatests.
* Move this test to `utils` or `fixtures` and spawn it as a separate process.

/cc @lundibundi

Reply:
@nechaido as a temporary workaround, I've moved the test to a different directory where each test file is then run sequentially in a separate process.